onnx/backend/test/case/node/ai_onnx_ml/tree_ensemble.py

```diff
@@ -0,0 +1,122 @@
+# Copyright (c) ONNX Project Contributors
+
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+
+import onnx
+from onnx.backend.test.case.base import Base
+from onnx.backend.test.case.node import expect
+from onnx.helper import make_tensor
+
+
+class TreeEnsemble(Base):
+    @staticmethod
+    def export_tree_ensemble_single_tree() -> None:
```

Comment:
I would add regressor in the test name. Another one must be added for a classifier.

Reply:
> Another one must be added for a classifier.

To implement a classifier using the `TreeEnsemble` kernel requires using some extra operators (`ArgMax`, `LabelEncoder` or `GatherND`). Are you suggesting a test that uses more than one operator here?

Reply:
Yes, let's add that in another PR.